presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -341,6 +340,111 @@ public void testColumnNameAmbiguity()
         session.execute("DROP KEYSPACE keyspace_5");
     }
 
+    @Test
+    public void testInsert()
+    {
+        String sql = "SELECT key, typeuuid, typeinteger, typelong, typebytes, /*typetimestamp,*/ typeansi, typeboolean, typedecimal, " +
+                "typedouble, typefloat, typeinet, typevarchar, typevarint, typetimeuuid, typelist, typemap, typeset" +
+                " FROM " + TABLE_ALL_TYPES_INSERT;
+        assertEquals(execute(sql).getRowCount(), 0);
+
+        // TODO Following types are not supported now. We need to change null into the value after fixing it
+        // blob, frozen<set<type>>, inet, list<type>, map<type,type>, set<type>, timeuuid
+        // timestamp, decimal, uuid, varint can be inserted but the expected and actual values are not same
```

Comment:
Sorry, I will update the comment. When I sent this PR at first, some types can be inserted with invalid values. But now, the behavior has been changed (maybe by datastax update?) and it can only insert null or not-null throws exception.